app/forms/signin_form.py

```diff
@@ -10,21 +10,37 @@ class SigninForm(FlaskForm):
     PASSWORD_LENGTH_MIN = 6
     PASSWORD_LENGTH_MAX = 20
 
-    email = StringField("Email", validators=[
-        DataRequired(message="Enter an email address."),
-        Length(max=EMAIL_LENGTH_MAX, message=f"Email cannot be more than {EMAIL_LENGTH_MAX} characters long."),
-        Email(message="Enter a valid email address.")
-    ])
-
-    password = PasswordField("Password", validators=[
-        DataRequired(message="Enter a password."),
-        Length(min=PASSWORD_LENGTH_MIN, max=PASSWORD_LENGTH_MAX, message=f"Password must be {PASSWORD_LENGTH_MIN} to {PASSWORD_LENGTH_MAX} characters long.")
-    ])
+    email = StringField(
+        "Email",
+        validators=[
+            DataRequired(message="Enter an email address."),
+            Length(max=EMAIL_LENGTH_MAX, message=f"Email cannot be more than {EMAIL_LENGTH_MAX} characters long."),
+            Email(message="Enter a valid email address.")
+        ]
+    )
+
+    password = PasswordField(
+        "Password",
+        validators=[
+            DataRequired(message="Enter a password."),
+            Length(
+                min=PASSWORD_LENGTH_MIN,
+                max=PASSWORD_LENGTH_MAX,
+                message=f"Password must be {PASSWORD_LENGTH_MIN} to {PASSWORD_LENGTH_MAX} characters long."
```

Comment:
Similar to above comment

Reply:
Fair so in the singin form we won't have any error message to do with length because they should already have a valid email / password of the appropriate length